README.md

```diff
@@ -154,73 +189,90 @@ Check if your Internet connection works
 $ ping -c3 www.archlinux.org
 ```
 
-If not see here
-([de](https://wiki.archlinux.de/title/Anleitung_f%C3%BCr_Einsteiger#Netzwerkverbindung_herstellen),
-[en](https://wiki.archlinux.org/index.php/Installation_guide#Connect_to_the_Internet))
-how to make it work or try first
+If you are not connected to the Internet and wish to use Wi-Fi, you'll need to
+identify your wireless network interface and connect using wpa_supplicant. Use
+the following command to list network interfaces and find your wireless network
+device:
+```bash
+$ ip a
+```
 
+Then connect through wpa_supplicant.
+Figure out you <wifi-name> with `ip a`.
 ```bash
-$ dhcpcd
+$ wpa_supplicant -B -i <wifi-name> -c <(wpa_passphrase "Your_SSID" Your_passphrase) && dhclient <wifi-name>
 ```
 
-Now configure the Mirrors, first backup the File
+For more details, consult the [official documentation](https://wiki.archlinux.org/title/Wpa_supplicant) 
+If you encounter issues or prefer a different method for connecting to the Internet, consult the
+[Arch Linux Installation Guide.](https://wiki.archlinux.org/index.php/Installation_guide#Connect_to_the_Internet)
+
+If you're facing issues with obtaining an IP address, you can manually trigger
+the DHCP client as follows:
 
-<details>
-  <summary>Old version</summary>
-  
-  For some reason, the old mirrorlist format isn't used anymore (2021.02.01) and therefore this code won't work anymore...
-  
-  ```bash
-  $ cp /etc/pacman.d/mirrorlist /etc/pacman.d/mirrorlist.bak
-  # XXX is your region e.g. Germany. use cat /etc/pacman.d/mirrorlist to see examples
-  $ grep -E -A 1 ".*XXX.*$" /etc/pacman.d/mirrorlist.bak | sed '/--/d' > /etc/pacman.d/mirrorlist
-  ```
-  
-</details>
+```bash
+$ dhcpcd
+```
+## Configure Mirror List (Optional)
```

Comment:
```suggestion
## Configure the mirror list (Optional)
```

Reply:
did put Configure the Mirror List and streamlined all the headings to uppercase except stuff like the...